nio/router/base.py

```diff
@@ -94,12 +99,18 @@ def configure(self, context):
         """
 
         self._mgmt_signal_handler = context.mgmt_signal_handler
+        self._instance_id = context.instance_id
         self._clone_signals = \
             context.settings.get("clone_signals", True)
         if self._clone_signals:
             self.logger.info('Set to clone signals for multiple receivers')
         self._check_signal_type = \
             context.settings.get("check_signal_type", True)
+        self._diagnostics = \
+            context.settings.get("diagnostics", True)
```

Comment:
I know we use it elsewhere, but how do these settings get properly converted to booleans when they are specified in files? I would think we would need to call a `getbool` on it at some point but don't know where that is happening

Reply:
context.settings is a dictionary built in nio-core which is where the Settings.getboolean call happens

Reply:
Aha! I see that PR now, just merged it. Makes sense, thanks